app/serializers/api/v2/serializable_course.rb

```diff
@@ -16,6 +16,10 @@ class SerializableCourse < JSONAPI::Serializable::Resource
         SubjectMapper.get_subject_list(@object.name, ucas_subjects)
       end
 
+      attribute :is_sen do
```

Comment:
I think this should stay as `is_send` because `SEN` isn't used on its own anymore with the `D`

Reply:
or `:is_send?`

Reply:
`is_send?` it will be then